aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js

```diff
@@ -63,36 +63,40 @@ export const FilteringCards = ({ containerRef }) => {
       <Heading level={2} margin="none">
         Users
       </Heading>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
-            data={users}
-            filters={filtersConfig}
-            layerProps={layerProps}
-            searchFilter={{ placeholder: 'Search' }}
-          />
+      <Box fill>
+        <Data
+          data={users}
+          updateOn="submit"
+          height={{ min: 'medium', max: '100%' }}
+        >
+          <Toolbar>
```

Comment:
It looks like in all of these we had to use `Toolbar` component instead of just `toolbar` prop on Data with `properties`. Is that the expected implementation most teams will have to follow? Just wondering because the current how to guide we have doesn't talk about this pattern.

Reply:
I thought the `toolbar` prop only provided a default implementation and didn't realize I could specify with `properties`. It's looking like using `toolbar` with Data is cleaner code, but I feel that we could still keep an example on the DS site that uses the Toolbar component. In other words, have the default behavior use `toolbar` with Data, then have an example show `Toolbar` where the user can be more specific?

Reply:
You might try using https://v2.grommet.io/data#properties instead.